src/evse_monitor.cpp

```diff
@@ -464,7 +484,7 @@ void EvseMonitor::setPilot(long amps, std::function<void(int ret)> callback)
     amps = _min_current;
   }
 
-  if(amps == _pilot)
+  if(amps == _pilot && !_pilot_is_wrong)
```

Comment:
Think it would be cleaner to add a parameter to the method with a default value, eg `bool force = false`

Reply:
ok I've added the param to setPilot